lib/representable/hash/collection.rb

```diff
@@ -19,6 +19,14 @@ def items(options={}, &block)
 
     # TODO: revise lonely collection and build separate pipeline where we just use Serialize, etc.
 
+    def from_hash(data, options={}, binding_builder=Binding)
+      data = filter_wrap(data, options)
+      data = [data] if ::Hash === data
```

Comment:
@apotonick this makes `#from_hash` more lenient - accepting a single Hash for a collection. I'm not sure if this is desirable.